### PR TITLE
Guncargo balance update 3: traitor edition

### PR DESCRIPTION
**Pull request by Zonespace27**
## About The Pull Request

- Riot pistol 2 -> 1
- Makarov 7 -> 5
- Revolvers 13 -> 12
- Riot SMG 5 -> 4
- Meteorslug & Slug Bulldog ammo 3 -> 2
- Bulldog 14 -> 12
- C-20R 13 -> 12

Added Energy shield module to uplinks for 12 TC

## How This Contributes To The Skyrat Roleplay Experience
Traitors are a tad on the back foot right now, these changes should make uplink guns a bit cheaper to buy, and the energy shield module should help them not get fucked up as badly. Changes aren't too drastic to reduce the dreaded *powercreep*.

## Changelog

:cl:
balance: Reduced cost of uplink firearms
add: Energy shield MODsuit module for 12 TC to uplinks
/:cl:

### CODE REVIEW

**Comment by Draganfrukts** [comment]
I think the makarov could be left at 5 TC, with the suppressor it's usually coupled with that leaves it at 7 TC

**Comment by YakumoChen** [comment]
The energy shield is kind of a noob trap given literally nobody is using energy weapons right now

**Comment by Zonespace27** [comment]
> The energy shield is kind of a noob trap given literally nobody is using energy weapons right now

MODsuit module energy shield blocks the first three projectiles that hit you, be it throwing, melee, bullets, energy, etc.

File: modular_skyrat/master_files/code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -284,15 +284,13 @@
 	desc = "An alternative 8-round meteorslug magazine for use in the Bulldog shotgun. \
 		Great for blasting airlocks off their frames and knocking down enemies."
 	item = /obj/item/ammo_box/magazine/m12g/meteor
-	cost = 3
 	purchasable_from = ALL
 
 /datum/uplink_item/ammo/shotgun/slug_traitor
```
Comment (YakumoChen) [inline modular_skyrat/master_files/code/modules/uplink/uplink_items.dm:289]: While you're here all these paths need to be in proper snake_case

File: modular_skyrat/master_files/code/modules/uplink/uplink_items.dm [hunk 2]
```diff
@@ -319,6 +317,13 @@
 	cost = 1
 	progression_minimum = 15 MINUTES
 
+/datum/uplink_item/suits/energy_shield_traitor
+	name = "MODsuit Energy Shield Module"
+	desc = "An energy shield module for a MODsuit. The shields can handle up to three impacts \
```
Comment (YakumoChen) [inline modular_skyrat/master_files/code/modules/uplink/uplink_items.dm:322]: Description needs to mention complexity costs

Comment (Zonespace27) [reply modular_skyrat/master_files/code/modules/uplink/uplink_items.dm:322]: Done

**Pull request merged**